apps/rush-lib/src/start.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { EOL } from 'os';
+import * as colors from 'colors';
+
+import rushVersion from './rushVersion';
+import RushCommandLineParser from './cli/actions/RushCommandLineParser';
+
+// tslint:disable-next-line:export-name
+export function executeCli(wrapperVersion: string, isManaged: boolean = false): void {
+  console.log(
+    EOL +
+    colors.bold(`Rush Multi-Package Build Tool ${rushVersion}` + colors.yellow(isManaged ? '' : '(unmanaged)')) +
```

Comment:
>unmanaged [](start = 98, length = 9)

Add a little code comment explaining that "(unmanaged)" is used to make it easier to recognize repos that don't lock down their Rush version, or cases where people invoke the tool outside of a repo.  It's mainly to help us when we see screenshots or log files